javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -576,7 +576,11 @@ public static Name getElementValueClassName(
             AnnotationMirror anno, CharSequence name, boolean useDefaults) {
         Name cn = getElementValueClassName(anno, name, useDefaults);
         try {
-            Class<?> cls = Class.forName(cn.toString());
+            ClassLoader appClassLoader = AnnotationUtils.class.getClassLoader();
```

Comment:
This is the same logic as contained in `AnnotationClassLoader.getAppClassLoader`.
Can we change that method to take as parameter for which to determine the classloader, make the method static, and invoke that method here?
Then we have one method that we should use in all places where `Class.forName` is invoked.